### ISSUE TITLE
Stage of released pretrained models

### ISSUE THREAD

**Issue by metric-space** [jule-c/flowr_root#5]
Hey @jule-c 

Goes without saying, `flowr-root` is pretty neat and thank you for the paper and the released models

I have a few questions about the released checkpoints and their associated training stages.

In the paper, you've mentioned three stages of training

1. `pretraining`
2. `finetuning`
3. `domain-adaptation`

For the pretrained models that you've released on gDrive, specifically [v2 checkpoint](https://drive.google.com/file/d/16dXF7C6iAFNpv1nT0W_N-6VtiqdDNWch/view?usp=drive_link)

1. What stages of training has it (v2 checkpoint) been passed through ?

2. Is it possible to go through stage 2 (`finetuning`) without going through stage 1 (`pretraining`)

3. Is it possible to go through stage 3 (`domain-adaptation`) without getting past stage 1 (`pretraining`) and/or  stage 2 (`finetuning`) ? 

In other words,  is "sequentialness" of your training process (`pretraining` -> `finetuning` -> `domain-adaptation`) a given much like the training workflow in the LLM world?

Can I confirm that the released pretrained models are past stage 1 and have neither been fine tuned nor domain adapted?

Thank you again

Luke

P.S. Hope it's alright to ask, do you have another model release planned for the near future?

**Comment by metric-space** [comment]
Hey @jule-c

Thank you so much for the detailed reply

> One nuance: domain adaptation is somewhat of an "enabler" in that without it, the model would fail completely on downstream tasks (like internal data)

I see. That among other things, gives me alot to think about

> However, it's not the fully trained version - neither completely pre-trained nor fully fine-tuned

I would not have guessed this was the case

Again thank you for the transparency and if it’s okay, I may reach out again with follow-up questions in the future

Best
Luke
